frontend/src/views/Pages/AccountOverview.vue

```diff
@@ -69,7 +69,7 @@ export default {
   data() {
     return {
       timestamp: Date.now(),
-      transactionData: EMPTY_TRANSACTION_DATA,
+      transactionData: { ...EMPTY_TRANSACTION_DATA },
```

Comment:
@ulfgebhardt Do you remember when you said this would be the same as
```
transactionData: EMPTY_TRANSACTION_DATA,
```
Well, it is NOT!

Reply:
How so? isn't `...` unwinding the object so you can rewind it again wit `{}`?  Whats the difference?

Reply:
@ulfgebhardt When you do `a = b` than `b` is the same as `a`. This means, when you change `b` you also change `a`. But when you say `a = { ...b }`, the content of `b` is copied to `a`. So when you change `b` it does not change `a`.